services/web/server/src/simcore_service_webserver/garbage_collector_core.py

```diff
@@ -395,8 +393,11 @@ async def remove_orphaned_services(
             continue
 
         project_uuid = resources["project_id"]
-        node_ids = await get_workbench_node_ids_from_project_uuid(app, project_uuid)
-        currently_opened_projects_node_ids[node_ids] = project_uuid
+        node_ids: set[str] = await get_workbench_node_ids_from_project_uuid(
```

Comment:
so this was failing since a long time, right?
We should get mypy in place ASAP

Reply:
As discussed offline, maybe we should activate this feature only on restricted subset of files, since it is hard to turn it on on the entire codebase of the `webserver`.